src/app/api/structures/structure.repository.ts

```diff
@@ -370,18 +379,17 @@ const createOrUpdateAdresses = async (
       // Update or create typologies
       for (const typologie of adresse.adresseTypologies || []) {
         // Update existing typologie
-        await prisma.adresseTypologie.upsert
-          ({
-            where: { id: typologie.id },
-            update: typologie,
-            create: {
-              adresseId: adresse.id,
-              placesAutorisees: typologie.placesAutorisees,
-              date: typologie.date,
-              qpv: typologie.qpv,
-              logementSocial: typologie.logementSocial,
-            },
-          });
+        await prisma.adresseTypologie.upsert({
+          where: { id: typologie.id },
```

Comment:
```suggestion
          where: { id: typologie.id || 0 },
```
Pas sûr pour cette modif ici, mais je l'ai vue ailleurs et je me demandais pourquoi elle n'était pas sur ce `upsert`